src/api/BioLink.js

```diff
@@ -540,48 +540,55 @@ function getBiolinkAnnotation(cardType) {
   return result;
 }
 
-export async function getNodeAssociations(
-  nodeType,
-  nodeId,
-  cardType,
-  taxons,
-  params = new URLSearchParams(),
-) {
+export async function getNodeAssociations(nodeType, nodeId, cardType, taxons, parms) {
```

Comment:
should parms be params?

Reply:
wait nevermind - see https://github.com/monarch-initiative/monarch-ui/pull/352/files#diff-ec1bbcbf802856ad9001d1b8c3b89ce3R556 now